lib/polyfill/vttcue.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.provide('shaka.polyfill.VTTCue');
+
+goog.require('shaka.log');
+goog.require('shaka.polyfill.register');
+
+/**
+ * @namespace shaka.polyfill.VTTCue
+ *
+ * @summary A polyfill to provide VTTRegion interface in VTTCue.
+ */
+
+
+/**
+ * Add the VTTRegion polyfill to VTTCue if needed
```

Comment:
It basically is used to specify the vttregion holding positioning info and is later used when positioning the actual text in browser that does not provide native VTTRegion support. Once all browsers support this the polyfill and the "special" positioning code is not needed